packages/tools/client-debugger/client-debugger-view/src/components/FluidTreeView.tsx

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import React from "react";
+import { HasContainerId, FluidObjectTreeNode } from "@fluid-tools/client-debugger";
+import { Accordion } from "./utility-components/";
+import { TreeDataView } from "./TreeDataView";
+
+/**
+ * {@link TreeView} input props.
+ */
+export interface FluidTreeViewProps extends HasContainerId {
```

Comment:
100% nitpick: We could introduce an interface `HasVisualTree<TNode extends VisualTreeNode>` to encapsulate the `node` property, and have our various props interfaces extend it. It would give us a common place to document `node` without having to document it redundantly everywhere.